core/os/device/remotessh/device.go

```diff
@@ -316,6 +306,7 @@ func GetConnectedDevice(ctx context.Context, c Configuration) (Device, error) {
 	}
 
 	connection, err := ssh.Dial("tcp", fmt.Sprintf("%s:%d", c.Host, c.Port), sshConfig)
+
```

Comment:
this is a crazy nit, but I think if we're going to add an empty line, it should go right after the error handling if-block. That seems to be our pattern.